code-of-conduct/README.md

```diff
@@ -18,8 +18,7 @@
 
 ## About these documents
 
-This document is adapted from guidelines written by the [Django Project](https://www.djangoproject.com/conduct/enforcement-manual/), which was itself based on the [Ada Initiative](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports) template and the [PyCon 2013 Procedure for Handling Harassment Incidents](https://us.pycon.org/2013/about/code-of-conduct/harassment-incidents/). Contributors to that document: Adam Obeng, Aleksandra Pawlik, Bill Mills, Carol Willing, Erin Becker, Hilmar Lapp, Kara Woo, Karin Lagesen, Pauline Barmby, Sheila Miguez, Simon Waldman, Tracy Teal, Malvika Sharan, Kari L. Jordan, Samantha Ahern. Additional language was added by [Otter Tech](https://otter.technology) from the [PyCon U.S. 2018 Code of Conduct](https://us.pycon.org/2018/about/code-of-conduct/) ([licensed CC BY 3.0](http://creativecommons.org/licenses/by/3.0/)). 
+This document is adapted from guidelines written for [The Carpentries Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html), which was expanded from [Django Project](https://www.djangoproject.com/conduct/enforcement-manual/) originally based on the [Ada Initiative](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports) template and the [PyCon 2013 Procedure for Handling Harassment Incidents](https://us.pycon.org/2013/about/code-of-conduct/harassment-incidents/). Contributors to the reference documents are: Adam Obeng, Aleksandra Pawlik, Bill Mills, Carol Willing, Erin Becker, Hilmar Lapp, Kara Woo, Karin Lagesen, Pauline Barmby, Sheila Miguez, Simon Waldman, Tracy Teal, Malvika Sharan, Kari L. Jordan, Samantha Ahern. Additional language was added by [Otter Tech](https://otter.technology) from the [PyCon U.S. 2018 Code of Conduct](https://us.pycon.org/2018/about/code-of-conduct/) ([licensed CC BY 3.0](http://creativecommons.org/licenses/by/3.0/)). 
+In 2018 and 2019, The Carpentries Code of Conduct was revised to add a summary, straightforward examples of both beneficial and unwanted behaviors, evaluating intent, reporting guidelines, an appeal process, the procedure for following up with a reportee, terminology, Code of Conduct incident response procedure, termed suspension checklist, expanded clauses for conflicts of interest, and committee membership agreement -- all of which have been adopted by the OBF Code of Conduct. Contributors of these revised documents are Ethan White, Kari L. Jordan, Karin Lagesen, Malvika Sharan, Samantha Ahern, and Simon Waldman.
```

Comment:
If you made the linebreak to introduce a paragraph break, you'd have to add an empty line.